pkg/bootstrap/platform/config.go

```diff
@@ -37,11 +37,18 @@ type Config struct {
 	Dataplanes uint16
 	// DataplaneType is the type of dataplane to create (envoy or go-based)
 	DataplaneType string
+
+	// LogLevel is the log level.
+	LogLevel string
 }
 
 const (
 	// DataplaneTypeEnvoy represents an envoy-type dataplane.
 	DataplaneTypeEnvoy = "envoy"
 	// DataplaneTypeGo represents a go-type dataplane.
 	DataplaneTypeGo = "go"
+	// InfoLogLevel represents an info log level.
+	InfoLogLevel = "info"
```

Comment:
I don't think we need these definitions. The logging level strings are pretty conventional.
Also, for envoy debugging TRACE level is much more helpful.
So I would use `trace` on `DEBUG=1`.